workspaces/confluence/.changeset/khaki-lizards-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-search-backend-module-confluence-collator': patch
```

Comment:
I suggest you bump to `minor` as your extending functionalities without betaking existing implementation